zerver/lib/actions.py

```diff
@@ -7795,9 +7800,12 @@ def do_resend_user_invite_email(prereg_user: PreregistrationUser) -> int:
 
     prereg_user.invited_at = timezone_now()
     prereg_user.save()
-    invite_expires_in_days = (
-        prereg_user.confirmation.get().expiry_date - prereg_user.invited_at
-    ).days
+
+    expiry_date = prereg_user.confirmation.get().expiry_date
+    if expiry_date is None:
+        invite_expires_in_days = None
+    else:
+        invite_expires_in_days = (expiry_date - prereg_user.invited_at).days
```

Comment:
Can we add a block comment noting that we're resetting the invitation to expire as long after the reminder is sent as it lasted originally?

Reply:
Done.